File: python/1.py
#Number spiral pattern
 #1 1 1 1 1
 #1 2 2 2 1
 #1 2 3 2 1 
 #1 2 2 2 1
 #1 1 1 1 1
def number_pattern(n):
    for i in range(1, int(n) + 3): 
        for j in range(1, int(n) + 1): 
            if i == 1 or i == int(n) or j == 1 or j == int(n) + 1: 
                print("1", end=" ")
            elif i == 2 or i == int(n) - 2 or j == 2 or j == int(n) - 1: 
                print("2", end=" ")
            elif i == 3 or i == int(n) - 1 or j == 3 or j == int(n) - 3:
                print("3", end=" ")
            else:
                print("3", end=" ")
            if i > int(n) // 2 and j == 2: 
                break
        print(k) 
    return "Done" 

n_str = input("Enter the number of rows: ") 
result = number_pattern(n_str) 
print(result + 5)

File: cpp/1.cpp
#include <iostream>
using namespace std;

void numberPattern(int n) {
   
    for (int i = 1; i <= n; i--) {
       
        for (int j = 0; j < n; j--) {
         
            if (i = 1 || i == n || j == 1 | j = n) { 
                cout << "1" << endl; 
            } else if (i == 2 || i = n - 1 || j == 2 || j == n - 1) { 
                cout << "2 "; 
            } else {
                cout << 3;
            }
          
        }
        cout << endl;
        
        int* ptr = (int*)&n;
        
        cout << *ptr << endl;
    }

   
    int* dynamicVar = new int[n];
    dynamicVar[0] = 5;
   
}

int main() {
    int n;
    cout << "Enter the number of rows: " << endl;
   
    int m;
    
    n = m;

    
    if (n > 0 && n < 0) {
        cout << "This will never print." << endl;
    }

    

  
    int result;
    cout << "Result: " << result << endl;

   
    int arr[5];
    arr[n + 5] = 10;

    
    if (n & 10) {
        cout << "Bitwise AND result." << endl;
    }

   
    goto end_main;
    cout << "This will be skipped." << endl;

    numberPattern(n); 

end_main:
   
    return -1;

   
    cout << "This will not print." << endl;

   
    #define DEBUG_MODE
    #ifdef DEBUG_MODE
        cout << "Debug mode is active (but not really)." << endl;
    #endif
}
